django/views/templates/technical_500.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
 <head>
   <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  <meta name="color-scheme" content="light dark" />
```

Comment:
```suggestion
  <meta name="color-scheme" content="light" />
```

Same as 404 – we don’t have support for a dark theme here, we should hard-code so the browser only ever renders a light theme.